src/commands/status.yml

```diff
@@ -26,6 +26,12 @@ parameters:
     description: >
       If `true`, notifications successful jobs will not be sent
 
+  change_only:
+    type: boolean
+    default: false
+    description: >
+      If `true`, notifications that are not "broke" or "fixed" will not be sent.
```

Comment:
can we change 'broke' to 'failed'?

Reply:
Sure, I think I used the terminology from another orb / iteration.